bin/loading-page.html

```diff
@@ -61,6 +61,11 @@ <h4 class="Footer-links-header"><a href="/en-GB/firefox/"><!-- react-text: 236 -
                 </a>
               </li>
               <li><a href="https://blog.mozilla.com/addons">Blog</a></li>
+              <li>
+                <a href="https://extensionworkshop.com/">
+                  Extension Workshop
+                </a>
+              </li>
               <li>
                 <a href="/en-GB/developers/">
                   <!-- react-text: 245 -->Developer Hub
```

Comment:
@willdurand what is this `<!-- react-text: 245` in this file? 🤔

Reply:
This file has been created with the output of the react app, hence weird HTML comments from react. It is useless but that page is useless too (only used for local dev when the server has not started yet)

Reply:
Ahh got it thanks for clarification William 👍